scanner/src/main/kotlin/voice/app/scanner/BookParser.kt

```diff
@@ -51,15 +51,31 @@ class BookParser
 
       val migratedPlaybackPosition = migrationSettings?.let { findMigratedPlaybackPosition(it, chapters) }
 
+      buildList {
+        add(null)
+        add("")
+      }.filterNotNull().firstOrNull()
       BookContent(
         id = id,
         isActive = true,
         addedAt = migrationMetaData?.addedAtMillis?.let(Instant::ofEpochMilli)
           ?: Instant.now(),
-        author = analyzed?.author,
+        author = analyzed?.artist,
         lastPlayedAt = migrationSettings?.lastPlayedAtMillis?.let(Instant::ofEpochMilli)
           ?: Instant.EPOCH,
-        name = migrationMetaData?.name ?: analyzed?.bookName ?: analyzed?.chapterName ?: file.bookName(),
+        name = buildList {
+          add(migrationMetaData?.name)
+          if (chapters.size == 1) {
```

Comment:
Does `chapters.size == 1` mean single file book? I see in some podcast episode mp3s, there are multiple chapters embedded in a single mp3. Those should also have title as the book name or may be '{album} - {tittle}'.

Reply:
@PaulWoitaschek 
```suggestion
          if (!file.isDirectory) {
```

Reply:
One chapter = one file already. Every chapter itself can contain chapter marks. 
It's not the best naming currently

Reply:
I see. Thanks for clarifying! Looking forward for the next release.